src/shared_state/practical.md

```diff
@@ -48,7 +48,7 @@ Uint8Array(20) [
 ]
 ```
 
-In some circumstances this format format could sent over the network in binary format, but [webxdc updates](http://localhost:3000/spec/sendUpdate.html) do not permit binary data in their payloads.
+In some circumstances this could sent over the network in binary format, but [webxdc updates](../spec/sendUpdate.html) do not permit binary data in their payloads.
```

Comment:
Could be sent over?

Reply:
it sounds natural to me, but that phrasing could be less common in other dialects. Maybe _across_ would work better in its place?

Reply:
and I double-checked that there aren't any other localhost links in the latest version